Makefile

```diff
@@ -0,0 +1,20 @@
+.PHONY: install clean lint test ci docs
```

Comment:
I thought I reviewed a PR where Kevin already made this change?

Reply:
I'm not sure what's going on there because '.PHONY' is in master / main